networkit/cpp/io/MatrixMarketReader.cpp

```diff
@@ -26,7 +26,7 @@ namespace {
 
 CSRMatrix MatrixMarketReader::read(const std::string &path) {
     std::ifstream in(path);
-    if (!in) {
+    if (!in.is_open()) {
```

Comment:
What's the warning that the static analyzer throws here?

Reply:
I don't remember the exact warning, the issue here is that we want to check if the file exists, but `if (!in)` does not do that.